server/labml_app/handlers.py

```diff
@@ -493,13 +494,13 @@ def sync_computer(computer_uuid: str) -> flask.Response:
         """
     c = computer.get_or_create(computer_uuid)
 
-    runs = request.args.get('runs', [])
+    runs = request.json.get('runs', [])
     if runs:
         res = c.sync_runs(runs)
 
         return utils.format_rv({'runs': res})
```

Comment:
If you are going to return here, might as well create two endpoints. One for long polling and one for syncing.

Reply:
done